zerver/lib/actions.py

```diff
@@ -547,17 +552,22 @@ def process_new_human_user(
     send_initial_pms(user_profile)
 
 
-def revoke_preregistration_users(
-    created_user_profile: UserProfile,
+def process_preregistration_users(
+    created_user: UserProfile,
```

Comment:
"process" is sufficiently general that it makes me think that this function doesn't really have a reason for existing separate from its one callsite in `process_new_human_user`.